### PR TITLE
used components CardContainer, NavContainer and Titel

### PR DESCRIPTION
**Pull request by Momcil**
used components CardContainer, NavContainer and Titel

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -3,13 +3,13 @@ import styled from 'styled-components'
 
 function Footer() {
   return (
-    <div className='Navbar'>
     <NavContainer>
+    <>
```
Comment (stefankorder) [inline src/components/Footer.jsx:7]: You dont need this fraction element, because u use allready your NavContainer

File: src/components/Card.jsx [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-        <h2>Morty Smith</h2>
-        <button>show more</button>
-      </div>
-  )
-}
-
-export default Card
\ No newline at end of file
+    <CardContainer>
+         <>
```
Comment (stefankorder) [inline src/components/Card.jsx:9]: You dont need this fraction element, because u use allready your CardContainer

**Review by stefankorder — CHANGES_REQUESTED** [review]
Pls fix typos too

File: src/components/Footer.jsx (hunk 1 above)
Comment (Momcil) [reply src/components/Footer.jsx:7]: tanks i will delete the unused fraction elements

**Pull request closed without merging**